transports/quic/src/transport.rs

```diff
@@ -27,6 +27,7 @@ use crate::endpoint::ToEndpoint;
 use crate::Config;
 use crate::{endpoint::Endpoint, muxer::QuicMuxer, upgrade::Upgrade};
 
+use futures::channel::mpsc::SendError;
```

Comment:
We already import the mpsc module, could remove this import by just saying `mpsc::SendError`.